mitosheet/mitosheet/tests/step_performers/test_userdefinedimport.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Copyright (c) Saga Inc.
+# Distributed under the terms of the GPL License.
+"""
+Contains tests for UserDefinedImport
+"""
+
+import pandas as pd
+import pytest
+from mitosheet.tests.test_utils import create_mito_wrapper
+
+USERDEFINEDIMPORT_TESTS = [
+    (
+        [
+            pd.DataFrame({'A': [1, 2, 3], 'B': [1.0, 2.0, 3.0], 'C': [True, False, True], 'D': ["string", "with spaces", "and/!other@characters"], 'E': pd.to_datetime(['12-22-1997', '12-23-1997', '12-24-1997']), 'F': pd.to_timedelta(['1 days', '2 days', '3 days'])})
+        ],
+        "put", 
+        "params", 
+        "here",
+        [
```

Comment:
These tests aren't actually testing the importer, right?

Reply:
We should add tests for 1) A custom import working 2) A custom import adding multiple files 3) A custom import that has a timeout in it and the sheet still working??

Reply:
Custom imports currently can only add one - otherwise, I added these tests. No timeout for now (slows down tests).